src/abstractdataframe/join.jl

```diff
@@ -416,9 +416,11 @@ function _join(df1::AbstractDataFrame, df2::AbstractDataFrame;
     end
 
     if indicator !== nothing
-        refs = left_indicator + right_indicator
-        pool = CategoricalPool{String,UInt8}(["left_only", "right_only", "both"])
-        indicatorcol = CategoricalArray{String,1}(refs, pool)
+        left_indicator .+= right_indicator
+        pa_base = PooledArray(["left_only", "right_only", "both"])
```

Comment:
Not used?
```suggestion
```

Reply:
right - sorry for forgetting to delete it.